api/edge_api/management/commands/ensure_identity_traits_blanks.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from typing import Any
+
+from django.core.management import BaseCommand
+
+from environments.dynamodb import DynamoIdentityWrapper
+
+identity_wrapper = DynamoIdentityWrapper()
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    def handle(self, *args: Any, **options: Any) -> None:
+        for identity_document in identity_wrapper.query_get_all_items():
+            should_write_identity_document = False
+            if identity_traits_data := identity_document.get("identity_traits"):
+                for trait_data in identity_traits_data:
+                    if "trait_value" not in trait_data:
+                        should_write_identity_document = True
+                        trait_data["trait_value"] = ""
+            if should_write_identity_document:
+                identity_wrapper.put_item(identity_document)
+                self.stdout.write(
+                    f"Fixed identity id={identity_document['identity_uuid']}",
+                )
```

Comment:
Maybe we should add some more log messages like this one to show some indication of progress?

I guess we don't want to write a message for every identity we iterate over, but perhaps we should add a counter, and log a message every 100k or something?

Reply:
If we had standardised stats export like Prometheus, sure.

At this point in time I prefer not to do this.

Besides, I don't expect that many corrupted identities.

Reply:
> Besides, I don't expect that many corrupted identities.

This is exactly the reason that I think we _should_ do this. Because it might be the case that, when it's running, it looks like nothing is happening.

Reply:
NB: I've decided to re-implement it using structlog.